ci/schema/mysql.sql

```diff
@@ -66,7 +66,8 @@ CREATE TABLE functional_alltypes (
     double_col DOUBLE,
     date_string_col TEXT,
     string_col TEXT,
-    timestamp_col TIMESTAMP,
+    -- sqlalchemy parses TIMESTAMP as non-nullable without the NULL modifier
```

Comment:
This appears to be [a "feature" of MySQL](https://dev.mysql.com/doc/refman/8.0/en/timestamp-initialization.html).

Reply:
ehh, good to know that :)